pkg/operator/encryptionprovider/encryptionprovider.go

```diff
@@ -67,37 +61,18 @@ func (p *encryptionProvider) EncryptedGRs() []schema.GroupResource {
 		return p.allEncryptedGRs // case 1 - we are in charge
 	}
 
+	// case 2 - CAO is in charge, reduce the list
```

Comment:
we could make the reduced list to be static too

Reply:
or have a subtract func